apis/r/R/QueryCondition.R

```diff
@@ -349,15 +359,18 @@ tiledbsoma_query_condition_from_triple <- function(
   invisible(qc)
 }
 
-# ================================================================
 #' Combine two 'tiledbsoma_query_condition' objects
 #'
 #' Combines two query condition objects using a relatiional operator.
 #'
-#' @param lhs A 'tiledbsoma_query_condition' object on the left-hand side of the relation
-#' @param rhs A 'tiledbsoma_query_condition' object on the right-hand side of the relation
-#' @param op_name A character value with the relation, which must be one of 'AND', 'OR' or 'NOT'.
-#' @param somactx SOMAContext pointer.
+#' @param lhs A 'tiledbsoma_query_condition' object on the left-hand side
+#' of the relation
+#' @param rhs A 'tiledbsoma_query_condition' object on the right-hand side
+#' of the relation
+#' @param op_name A character value with the relation, which must be one of
+#' 'AND', 'OR' or 'NOT'
```

Comment:
```suggestion
#' 'AND', 'OR', or 'NOT'
```

Let's consistently use the Oxford comma.